var/spack/repos/builtin/packages/openssh/package.py

```diff
@@ -81,6 +84,8 @@ def determine_version(cls, exe):
     def patch(self):
         # #29938: skip set-suid (also see man ssh-key-sign: it's not enabled by default)
         filter_file(r"\$\(INSTALL\) -m 4711", "$(INSTALL) -m711", "Makefile.in")
+        # #39599: fix configure to parse zlib 1.3's version number to prevent build fail
+        filter_file(r"if \(n != 3 && n != 4\)", "if (n < 2)", "configure")
```

Comment:
Upstream they use https://github.com/openssh/openssh-portable/commit/cb4ed12ffc332d1f72d054ed92655b5f1c38f621 < 1 :thinking:

Reply:
I guess you're using `filter_file` cause patches may need to be done on a per version basis, but I still prefer patch files since they're more reliable.

Reply:
Bad, I tested it with all many versons across the board. I'm on vacation now, and can't update the PR 

Take it or let the buid failures accumulate.